test/fixtures/organisations.yml

```diff
@@ -54,7 +54,7 @@ rails_world:
   frequency: 1
   youtube_channel_id: UC9zbLaqReIdoFfzdUbh13Nw
   youtube_channel_name: railsofficial
-  slug: railsworld
+  slug: rails-world
```

Comment:
We probably need to do something manually in prod for that change

Reply:
I actually already changed that months ago:

https://www.rubyvideo.dev/organisations/rails-world